whipper/src/main/java/org/whipper/WhipperProperties.java

```diff
@@ -485,7 +485,8 @@ public String getQuerySetDir(){
      * @return expected results directory name
      */
     public String getExpectedResultsDir(){
-        return getProperty(Keys.EXPECTED_RESULTS_DIR);
+        String[] trimmedExpectedResultsDirArray = getProperty(Keys.EXPECTED_RESULTS_DIR).trim().split("\\s*,\\s*");
```

Comment:
Could you please add description of new form of values to "example_scenario_file.properties"?
Do not forget that order is significant and which entry takes precedence.